src/status_im/ui/screens/chat/stickers/views.cljs

```diff
@@ -76,6 +78,9 @@
     (when pack-id
       (re-frame/dispatch [:stickers/select-pack pack-id]))))
 
+(defn- valid-sticker? [sticker]
```

Comment:
Maybe this fn can be dropped in favour of `stickers.core.valid-stickers?`

Reply:
:+1: